plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -970,7 +968,7 @@ func TestAdmitSeccomp(t *testing.T) {
 		pod                   *kapi.Pod
 		psps                  []*extensions.PodSecurityPolicy
 		shouldPass            bool
-		expectedPodAnnotation string
+		expectedPodProfile string
```

Comment:
Yeah, I'll reformat.  Just getting the actual conversions up here for you to take a look at.